src/test/java/com/iota/iri/NodeIntegrationTests.java

```diff
@@ -135,7 +135,7 @@ Runnable spawnCoordinator(API api, long spacing) {
     }
 
     private void sendMilestone(API api, long index) throws Exception {
-        newMilestone(api, api.getTransactionToApproveStatement(10, null, 1), index);
+        newMilestone(api, api.getTransactionToApproveStatement(10, null, 1, new ArrayList<>()), index);
```

Comment:
Collections.EMPTY_LIST will be better.
Saves memory and IIUC the list is supposed to be immutable